invenio_accounts/version.py

```diff
@@ -28,4 +28,6 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = "0.2.1.dev20151005"
+from __future__ import absolute_import, print_function
```

Comment:
No problem. It is just that any of the directives is needed in this specific file.